posthog/api/session_recording.py

```diff
@@ -97,7 +99,12 @@ def list(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
     def retrieve(self, request: request.Request, *args: Any, **kwargs: Any) -> response.Response:
         session_recording_id = kwargs["pk"]
         filter = SessionRecordingsFilter(request=request)
-        session_recording_meta_data = self._get_session_recording_meta_data(request, filter, session_recording_id)
+
+        include_active_segments = True if request.GET.get("include_active_segments") else False
```

Comment:
If request.GET.get("include_active_segments") == "False" this will still include active segments

Reply:
Good catch!